docs/Setup-Dev.md

```diff
@@ -0,0 +1,27 @@
+#Software to install:
+-[Visual Studio](https://visualstudio.microsoft.com/it/thank-you-downloading-visual-studio/?sku=Community&rel=16)
+-[Visual Studio Code](https://code.visualstudio.com/)
+-Client Git([Fork](https://git-fork.com/) o [GitExtensions](http://gitextensions.github.io/))
+-[NodeJS last LTS](https://nodejs.org/it/download/)
+-[MongoDB Server](https://www.mongodb.com/) 
+-[MongoDB Compass](https://www.mongodb.com/) (optional)
+-[Postman](https://www.getpostman.com/downloads/) (optional)
+
+#Configuration
+##Visual Studio's port setting the same as Postman
+On Postman import the collections: open the folder of rawCMS->docs->RawCMS.postman_collection.json.
```

Comment:
replace this line with:
On Postman import the collection file `rawCMS/docs/RawCMS.postman_collection.json`

Reply:
use the line of this comment